clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -138,7 +138,7 @@ public synchronized boolean updateRequested() {
     /**
      * Wait for metadata update until the current version is larger than the last version we know of
      */
-    public synchronized void awaitUpdate(final int lastVersion, final long maxWaitMs) throws InterruptedException {
+    public synchronized Cluster awaitUpdate(final int lastVersion, final long maxWaitMs) throws InterruptedException {
```

Comment:
@ijuma Thank you for the review. You are right, this is an unnecessary change. No idea what I was thinking.